<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>about</class>
 <widget class="QDialog" name="about">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>220</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; font-style:italic; color:#ff0000;&quot;&gt;版权所有，盗版必究！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="movieLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="yesButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>YES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>NO</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>movieLabel</zorder>
  <zorder>yesButton</zorder>
  <zorder>noButton</zorder>
  <zorder>noButton</zorder>
  <zorder>noButton</zorder>
  <zorder>yesButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
